SNDINFO.hitmarker

```diff
@@ -0,0 +1,47 @@
+//Hit Marker Sound Defs
+
+// Call of Duty: Black Ops 4
+blackops4hit		HMBO4H
+blackops4kill		HMBO4K
+
+// Call of Duty: Black Ops Cold War
+blackopscoldwarhit	HMBOCWH
+blackopscoldwarkill	HMBOCWK
+
+// Call of Duty 4
+callofduty4hit		HMCOD4H
+
+// Default
+defaulthit			HMDEFH
+defaulthithigh		HMDEFHH
```

Comment:
Notice how messy tabs look
They will look much better if you'll replace them with 4 spaces

There is probably tab width setting somewhere in github but in my experience you'll have much less hassle if you'll use spaces instead of tabs.

Related VSCode settings (File->Preferences->Settings->Search settings):

- "Render Whitespace": controls how VSCode renders whitespace
Mine is set to "all". Very handy.
 
- "Detect indentation": tries to guess if current file is using tabs or spaces for indentation. Annoying setting.
Mine is turned off, default is on

- "Tab size": sets tab width in spaces
Mine is set to 4, I think that's default

- "Insert spaces": inserts spaces instead of tabs based on "Tab size" setting
Mine is turned on